packages/common-algorand/package.json

```diff
@@ -14,7 +14,7 @@
   "main": "dist/index.js",
   "license": "GPL-3.0",
   "dependencies": {
-    "@subql/common": "^3.5.0",
+    "@subql/common": "^3.5.1",
```

Comment:
I think we missed update types `project/utils.ts`

Reply:
They already seem to have the right types